Refactor APM getting started experience (#4661)
DataDog/documentation: content/en/tracing/setup/_index.md

---

> title: Configure your application to collect traces

Reviewer: ```suggestion
title: Configure your application to emit traces
```

kind: documentation
aliases:
    - /tracing/languages
    - /tracing/proxies
disable_toc: true
---

> After you have [installed the Datadog Agent][1], and [enabled trace collection][2], then configure your application to send traces using one of the libraries locally, or in [a container][3].

Reviewer: ```suggestion
After you have [enabled trace collection][2], configure your application to send traces using one of the following official Datadog tracing libraries:
```

Reviewer: removing the Agent reference since it's only one of several ways to collect traces.

### Language setup

{{< partial name="apm/apm-languages.html" >}}

To instrument an application written in a language that does not yet have official library support, visit the list of [community tracing libraries][4].

[1]: /agent

> [2]: /agent/apm

Reviewer: ```suggestion
[2]: /tracing/enable/
```
